packages/grafana-schema/src/schema/dashboard/v2alpha0/types.gen.ts

```diff
@@ -871,6 +871,43 @@ export const defaultResponsiveGridLayoutItemSpec = (): ResponsiveGridLayoutItemS
 	element: defaultElementReference(),
 });
 
+export interface TabsLayoutKind {
```

Comment:
@dprokop This is a very good example of where we could use status. We can save the current tab index. This will probably be desired as we save the current time range or variables selected.

Could we try to use this as a first usage of status for a resource? Otherwise, I think we should add to the spec `currentTab`  defaulting to `0`

Reply:
@ivanortegaalba I'm not sure we are able to save the status yet. Other than that I think we need to land on decision whether or not we want to use it for such things. I.e. why would we save the current tab index in the first place? I would rather imagine tab layout runtime to default to tab 0, and other than that current tab should be inferred from URL.